src/migrations/005-other-settings-vuex-to-composables.ts

```diff
@@ -8,6 +8,7 @@ const migration: Migration<IOtherSettings> = async (restoredValue: IOtherSetting
     return {
       isSeedBackedUp: backedUpSeed,
       saveErrorLog,
+      secureLoginTimeout: IS_MOBILE_APP ? AUTHENTICATION_TIMEOUTS[0] : AUTHENTICATION_TIMEOUTS[5],
```

Comment:
Migrations should only by done when:
1. The key has changed
2. The vale should be recalculated.

In this none of the rules applies.